server/infra/database/BaseRepository.spec.js

```diff
@@ -4,15 +4,15 @@ const knex = require("../database/knex");
 const mockKnex = require("mock-knex");
 const tracker = mockKnex.getTracker();
 const jestExpect = require("expect");
-const Session = require("../models/Session");
+const Session = require("./Session");
 
 describe("BaseRepository", () => {
   let baseRepository;
 
   beforeEach(() => {
     mockKnex.mock(knex);
     tracker.install();
-    const session = new Session();
+    const session = new Session(false);
```

Comment:
I was trying to run unit tests but they are failing currently, have asked the person who wrotes about the reason for the failure (caused by a missing `testTable` table). Please ignore this edit on the file